prov/verbs/src/ep_rdm/verbs_ep_rdm.c

```diff
@@ -76,8 +77,13 @@ fi_ibv_rdm_tagged_find_max_inline_size(struct ibv_pd *pd,
 			ibv_destroy_qp(qp);
 		qp_attr.cap.max_inline_data = max_inline;
 		qp = ibv_create_qp(pd, &qp_attr);
-		if (qp)
+		if (qp) {
+			/* 
+			 * truescale returns max_inline_data 0
+			 */
+			if (0 == qp_attr.cap.max_inline_data) break;
```

Comment:
Can you put the break on a separate line, and reverse the == 0 check?  Single line if statements hide statements, and reverse logic is confusing to read.